MTM-Backend/src/index.ts

```diff
@@ -34,8 +34,11 @@ app.use("/example", verifyToken, exampleRoute);
 // app.use("/example", exampleRoute);
 
 app.use("/users", verifyToken, userRouter);
-app.use("/items", verifyToken, itemsRouter);
 // app.use("/users", userRouter);
+
+// app.use("/items", verifyToken, itemsRouter);
```

Comment:
Pls add back the verifyToken middleware when merging with main

Reply:
Sorry, my bad.